source/cloud/cloud-administration/converting-oauth-2.0-to-openid-connect.rst

```diff
@@ -0,0 +1,26 @@
+Converting OAuth 2.0 to OpenID Connect
+=======================================
+
+OAuth 2.0 is being deprecated and replaced by OpenID Connect. You need to migrate your OAuth configuration to the new OpenID Connect standard. 
+[within what timeframe?] [failure to do so will result in....tbd]
```

Comment:
@wiersgallak @sbishel Can you provide some details about the urgency around converting, please? Is there a deadline that should be adhered to? Are there consequences to System Admins not converting ASAP?

Reply:
Thanks! I've adjusted the tone of the sentence to feel less urgent/time sensitive.